rules/concepts.md

```diff
@@ -194,8 +198,504 @@ Available types of conditions include:
+## Scripts
+
+Unfortunately the term "Script" is overloaded in openHAB, and has multiple meanings based on the context:
+
+- A UI rule consisting only of a single Script Action with the tag "Script".
+  This can be used to reuse code and logic across multiple rules.
+- In text based rules, a script is a file that is processed by a script engine, e.g. [JS Scripting](/addons/automation/jsscripting/).
+  Those scripts can be used to create (multiple) rules, or those are loaded by other script files a libraries.
+- A script file that is executed on the command line with a script engine, e.g. `bash`, by using `executeCommandLine`.
+- A special piece of Rules DSL code in the `$OH_CONF/scripts` folder that is called from other rules using `callScript`.
+
+The rule docs mainly refer to the first two meanings when talking about scripts.
+
+## Script Actions
+
+Script Actions are a special type of action only available in UI rules.
+
+These allow you to run scripts written in one of the available automation/rule languages, e.g. JavaScript, Rules DSL, and Python.
+
+## Available Values
+
+When a rule is triggered, some information about the event that triggered the rule is provided to the rule.
+There are many cases in which it is useful to know what triggered your rule, e.g. you have an Item group as trigger and you need to know which Group member triggered the rule.
+
+The availablity of those values depends on the rule engine, but you can generally expect at least the following information (depending on the trigger):
+
+- The name of triggering Item.
+- The command that the triggering Item received.
+- The state the Item got updated by.
+
+For further information which values are available and how to access them, refer to the documentation of the according rule engine.
+
+## Rule Templates
+
+At some point, when basic UI rules are not be sophisticated enough to reach your goal, rule templates can help.
+Someone may have already written the rule and provides it as a template at the community marketplace.
+
+To access the rule templates, visit the web UI and navigate: *Settings* -> *Add-Ons* -> *Automation* -> Scroll down to the *Rule Templates* section.
+Click on a rule template to review it's documentation, open the community thread and install the template to your system.
+Check the template's documentation for dependencies and install them!
+Otherwise the rule won't work.
+
+To instantiate a rule template, navigate to Rules and click the blue `+` icon.
+Fill out the rule's metadata as usual and select a installed rule template from the *Create from Template* section.
+Choose the rule template, and fill out the template configuration.
+
+Once a rule is created, it is seperated from it's templated and can be modified individually.
+To update rule from the template, you have to recreate the rule.
+
+If you want to create your own rule templates, visit the [Marketplace](https://community.openhab.org/c/marketplace/rule-templates/74) at the community forum.
```

Comment:
I think we should move that content to the UI Rules page we are about to create as part of this effort. It belongs in the main docs I think.

Reply:
I‘ve moved it to the UI Rules page as a comment.